File: app/src/main/java/com/comicsopentrends/fragments/mvp/clans/presenter/CharactersFragmentPresenter.java
package com.comicsopentrends.fragments.mvp.clans.presenter;

/**
 * Created by Asus on 20/10/2017.
 */

public interface CharactersFragmentPresenter {

    void searchCharacter(String query);

    void loadList();

    void onLoadMore();

    void resetVariables();

    void onRefresh();
}

File: app/src/main/java/com/comicsopentrends/fragments/mvp/clans/presenter/CharactersFragmentPresenter.kt
package com.comicsopentrends.fragments.mvp.clans.presenter

/**
 * Created by Asus on 20/10/2017.
 */

interface CharactersFragmentPresenter {

    fun searchCharacter(query: String)

    fun loadList()

    fun onLoadMore()

    fun resetVariables()

    fun onRefresh()
}
